observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,6 +12,109 @@ You can use the {TempoName} xref:../../observability/otel/otel-forwarding-teleme
+
+[NOTE]
+====
+The {JaegerName} 3.5 is based on the open source link:https://www.jaegertracing.io/[Jaeger] release 1.62.0.
+====
+// 1.62.0 is fixed until Jaeger's removal
+
+[IMPORTANT]
+====
+Jaeger does not use FIPS validated cryptographic modules.
+====
+
+[id="distr-tracing_3-5_jaeger-release-notes_support-for-elasticsearch-operator_{context}"]
+==== Support for the {es-op}
+
+The {JaegerName} 3.5 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+
+[id="distr-tracing_3-5_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.5, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red{nbsp}Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
+
+The {JaegerOperator} Operator (Jaeger) will be removed from the `redhat-operators` catalog in a future release. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/7083722[Jaeger Deprecation and Removal in OpenShift].
+
+You must migrate to the {OTELName} Operator and the {TempoOperator} for distributed tracing collection and storage. For more information, see xref:../../observability/otel/otel-migrating.adoc#dist-tracing-otel-migrating[Migrating] in the {OTELShortName} documentation, xref:../../observability/otel/otel-installing.adoc#install-otel[Installing] in the {OTELName} documentation, and xref:../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Installing] in the {TempoShortName} documentation.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/solutions/7083722[Jaeger Deprecation and Removal in OpenShift] (Red Hat Knowledgebase)
+* xref:../../observability/otel/otel-migrating.adoc#dist-tracing-otel-migrating[Migrating] ({OTELName} documentation)
+* xref:../../observability/otel/otel-installing.adoc#install-otel[Installing] ({OTELName} documentation)
+* xref:../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Installing] (Distributed tracing platform (Tempo) documentation)
+
+////
+[id="distr-tracing_3-5_jaeger-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fix:
+
+* ???
+
+////
+
+[id="distr-tracing_3-5_jaeger-release-notes_known-issues_{context}"]
+==== Known issues
+
+There are currently known issues:
+
+* Currently, Apache Spark is not supported.
+
+ifndef::openshift-rosa[]
+
+* Currently, the streaming deployment via AMQ/Kafka is not supported on the {ibm-z-title} and {ibm-power-title} architectures.
```

Comment:
IMO this isn't a merge blocker but IBM Style doesn't like "via." (for future docs)

Reply:
Thank you for the reminder. I checked again, and we're on good terms with IBM Style here:
```
Use this term, which means by way of, to refer to routing data, such as in “Messages are routed via SMTP”.
```